src/controller/python/chip-device-ctrl.py

```diff
@@ -479,6 +479,8 @@ def do_connect(self, line):
             elif args[0] == "-ble" and len(args) >= 3:
                 self.devCtrl.ConnectBLE(int(args[1]), int(args[2]), nodeid)
             elif args[0] == '-qr' and len(args) >=2:
+                if len(args) == 3:
+                    nodeid = int(args[2])
```

Comment:
Why don't we just use argparse?  This looks like a lot of unnecessary boilerplate.

Reply:
I'm not sure - I didn't write the original code. Agree it could use a refactor, but suggest we defer that until after TE3.